.github/workflows/community-releases.yml

```diff
@@ -17,52 +12,101 @@ permissions:
 jobs:
   prepare:
     runs-on: ubuntu-latest
-    timeout-minutes: 80
+    timeout-minutes: 60
+    outputs:
+      lerna-release-version: ${{ steps.get_version.outputs.lerna-release-version }}
+      check_tag_result: ${{ steps.check_tag.outputs.result }}
     steps:
       - uses: actions/checkout@v4
+      - name: Get Current Lerna Release version
+        id: get_version
+        shell: bash
+        run: |
+          echo "lerna-release-version=$(jq -r '.version' lerna.json)" >> $GITHUB_OUTPUT
+
+      - name: Check if tag already exists
+        id: check_tag
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const { data: tags } = await github.rest.git.listMatchingRefs({
```

Comment:
Nit-picky one: We can get the remote tag using simple git and avoid using github-script

```
git ls-remote --tags origin | grep $release-version
or
git show-ref --tags --verify --quiet "refs/tags/$release-versiom}"
```